src/app/home/pages/PlanDebugPage/PlanDebugPage.tsx

```diff
@@ -94,7 +105,7 @@ export const PlanDebugPage: React.FunctionComponent = () => {
               }}
             >
               <Title headingLevel="h1" size="xl" className={spacing.mrLg}>
-                Migration plan resources
+                Migration resources
```

Comment:
This was a change we discussed with Vince, since now the view is isolated to those relevant to a single migration.